chpl/chpl-service/src/test/java/gov/healthit/chpl/manager/impl/SearchMenuManagerTest.java

```diff
@@ -593,7 +593,7 @@ public void test_getEditionNames_CacheRefreshesWithParameter()
     @Transactional
     @Test
     public void testGetNotificationTypesForAdminUser() {
-        final int expectedCount = 1;
+        final int expectedCount = 8;
```

Comment:
I think this should be back to 1. The notification types are down to just questionable activity right now